.travis.yml

```diff
@@ -35,6 +35,7 @@ before_install:
  - HC=${CC}
  - unset CC
  - PATH=/opt/ghc/bin:/opt/ghc-ppa-tools/bin:$PATH
+ - if [ $GHCVER = "7.4.2" ]; then mv cabal.ghc-7.4.2.project cabal.project; fi
```

Comment:
This most likely isn't needed and we should avoid it. While Stack may have problems coping with that configuration I honestly doubt that Cabal has any issues with this. Have you run into an actual problem? Also it'd defeat the purpose of CI if we exclude configurations even though they're declared to be compatible via `build-depends`.